src/go/rpk/pkg/cli/cluster/selftest/status.go

```diff
@@ -164,6 +164,11 @@ func makeReportTable(report adminapi.SelfTestNodeReport) [][]string {
 			table = append(table, []string{""})
 			continue
 		}
+		if sr.TestType == adminapi.CloudcheckTagIdentifier {
+			// Cloudcheck does not have any IOPS, THROUGHPUT, or LATENCY results.
+			table = append(table, []string{""})
```

Comment:
nit: it'd be helpful as a reviewer if either in the commit message or the cover letter you pasted some sample output